src/sentry/integrations/github_enterprise/webhook.py

```diff
@@ -122,7 +122,7 @@ def handle(self, request: Request) -> HttpResponse:
             host = get_host(request=request)
             if not host:
                 logger.warning("github_enterprise.webhook.missing-enterprise-host")
-                logger.exception("Missing enterprise host.")
+                logger.error("Missing enterprise host.")
```

Comment:
@vartec why does this become `logger.errror` when we are switching the other ones to `logger.exception`?

Reply:
This one is _not_ inside an exception handler, therefore it should be be an error per https://github.com/adamchainz/flake8-logging#log004-avoid-exception-outside-of-exception-handlers

On the other hand, when inside an exception handler, exception should be used instead of an error per https://github.com/adamchainz/flake8-logging#log005-use-exception-within-an-exception-handler